src/components/button/Button.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import styles from './Button.css';
+
+class Button extends Component {
```

Comment:
@paulobmarcos asked this in the first review, and I replied the reason I did not change it to his comment:

As of this time, the button will remain a class component. It will be changed to a functional component when TooltipTrigger receives the changes requested in https://github.com/ipfs-shipyard/pm-idm/issues/78 .

Summing up, the tooltip trigger, with its current implementation, doesn't allow a functional component to be a trigger. I don't feel that breaking the behavior of tooltip (until the referred task is completed) should be done just to make the button a stateless component.

There is an added reason, which I found this afternoon. The next version of the button will need to be stateful to allow for the loading animation, so I think we should leave it as is because it would be changed soon anyway.